salt/modules/file.py

```diff
@@ -4633,6 +4634,10 @@ def manage_file(name,
         a local file on the minion), the mode of the destination file will be
         set to the mode of the source file.
 
+    allow_unicode : False
+        if ``True``, file can have unicode characters and will be encoded
```

Comment:
I think it would be good to add `.. versionadded:: Nitrogen` to make it clearer when this feature was firstly introduced.